API/api/services/event_service.py

```
@@ -1,14 +1,46 @@
     def __init__(self):
         self.repo = EventRepository()
 
-    def create_event(self, data: dict) -> dict:
+from datetime import date
+
+def update_event(self, event_id: int, name: str | None, event_date: date | None):
+
+    event = self.repo.get_event_by_id(event_id)
+
+    if not event:
+        raise NotFound("Événement introuvable.")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)